src/internals.js

```diff
@@ -96,9 +96,16 @@ function parseConfig(config = {}) {
     return config;
 }
 
+function collect(val, memo) {
+    memo.push(val);
+
+    return memo;
+}
```

Comment:
why not just inline as `(val, memo) => { memo.push(val); return memo; }`? :)

anyway, doesn't hurt, just surprised ^^